### PR TITLE
LRDOCS-6440, installing elasticsearch

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6440, Intyroduction articles for Installing a Search Engine and Elasticsearch sections were ported from 7.1, and then the Installing Elasticsearch article was also ported and moved into this section. All steps were tested, no u[pdates were needed. Added some logging information and a screenwhot of the search engine info from Search Admin (used master for this).

### CODE REVIEW

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+[TOC levels=1-4]
+
+Elasticsearch is an open source, highly scalable, full-text search and
+analytics engine.
+
+By default, Elasticsearch runs as an embedded search engine, but it's only
+supported in production as a separate server or cluster. This guide walks you
+through the process of configuring Elasticsearch in remote mode.
+
+![Figure 1: To see information about the currently connected search engine, go to _Control Panel &rarr; Configuration &rarr; Search_. ](../../../images/search-admin-engineinfo.png)
```
Comment (lipusz) [inline deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: Hey Russ,

This image with some explanation should also be added to the end of 03-installing-elasticsearch.markdown as a sort of visual verification admins can use to check if Liferay is now really connecting to a remote ES server.

What do you think? @rbohl

**Comment by lipusz** [comment]
**Started reviewing...** :mag:

**Comment by lipusz** [comment]
That's my only remark. Thank you!

File: deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown (hunk 1 above)
Comment (rbohl) [reply deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown]: Agree - added it

**Comment by rbohl** [comment]
Forwarded: https://github.com/sez11a/liferay-docs/pull/4028

**Pull request closed without merging**